app/controllers/patients_controller.rb

```diff
@@ -0,0 +1,52 @@
+class PatientsController < ApplicationController
+  before_action :set_patient, only: %i[show update destroy]
+
+  # GET /patients
+  def index
+    @patients = Patient.all
+
+    render json: @patients
+  end
+
+  # GET /patients/1
+  def show
+    render json: @patient
+  end
+
+  # POST /patients
+  def create
+    @patient = Patient.new(patient_params)
+
+    if @patient.save
+      render json: @patient, status: :created, location: @patient
+    else
+      render json: @patient.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /patients/1
+  def update
+    if @patient.update(patient_params)
+      render json: @patient
+    else
+      render json: @patient.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /patients/1
+  def destroy
+    @patient.destroy!
+  end
```

Comment:
- [ ] What happens here if the patient is not successfully destroyed?  Kindly, add some error handling like you did in the #update method (line 28). 👍